dealership/contracts.py

```diff
@@ -1,12 +1,39 @@
 class Contract(object):
-    pass
+    def __init__(self, vehicle, customer):
+        self.vehicle = vehicle
+        self.customer = customer
+
 
 
 class BuyContract(Contract):
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
+        # super().__init__(vehicle, customer)
+        super(BuyContract, self).__init__(vehicle, customer)
+        self.monthly_payments = monthly_payments
+
+    
+    def total_value(self):
+        tv = self.vehicle.sale_price() + (self.vehicle.get_interest_rate() * self.monthly_payments * self.vehicle.sale_price() / 100)
+        if self.customer.is_employee():
+            return tv*0.9
+        else:
+            return tv
 
+    def monthly_value(self):
+        return self.total_value()/self.monthly_payments
 
 class LeaseContract(Contract):
     def __init__(self, vehicle, customer, length_in_months):
-        pass
+        # super().__init__(vehicle, customer)
+        super(LeaseContract, self).__init__(vehicle, customer)
+        self.length_in_months = length_in_months
+        
+    def total_value(self):
+        tv = self.vehicle.sale_price() +  self.vehicle.sale_price()*self.vehicle.get_lease_multiplier()/self.length_in_months
```

Comment:
In statements with operations, it's considered best practice to follow PEP8 conventions for whitespace and formatting. Have a look at https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements
Also, you want to avoid lines that are longer than 79 characters.